src/careamics_restoration/bioimage/rdfs/n2v.yaml

```diff
@@ -0,0 +1,29 @@
+name: Noise2Void
+description: Self-supervised denoising.
+license: BSD-3-Clause
+authors:
+- {name: Alexander Krull}
+- {name: Tim-Oliver Buchholz}
+- {name: Florian Jug}
+cite:
+- {doi: 10.48550/arXiv.1811.10980, text: 'A. Krull, T.-O. Buchholz and F. Jug, "Noise2Void
+    - Learning Denoising From Single Noisy Images," 2019 IEEE/CVF Conference on Computer
+    Vision and Pattern Recognition  (CVPR), 2019, pp. 2124-2132'}
+input_axes: ["bcyx"]
+preprocessing:
+  -
+    - kwargs:
+        axes: yx
+        mean: [46912.574]
+        mode: fixed
+        std: [16847.809]
+      name: zero_mean_unit_variance
+output_axes: ["bcyx"]
+postprocessing:
+  -
```

Comment:
is the single "-" a typo?

Reply:
No, that's because the `postprocessing` and the `preprocessing` must be a list of actual processing, so we can have more than one pre/post processings.